main.py

```diff
@@ -148,9 +149,16 @@ def emulate(self):
             self.emulating_proccess.terminate()
             self.emulating_proccess = None
         monitor = self.monitors[self.active_monitor]
+        flip_args = ""
```

Comment:
At first, in this project I use single-quote.
At second, we can write it a bit more efficiently
```python
flip_args = ''
if self.flip in ('h', 'v',): # We should have validation for options
    flip_args = f'-vf {self.flip}flip'
```